Sources/App/Github/GithubCommandRouter.swift

```diff
@@ -0,0 +1,42 @@
+import Vapor
+
+// Basic router implementation
+public class GithubCommandRouter {
+    /// The internal router
+    private let router: TrieRouter<Responder>
+    
+    public var routes: [Route<Responder>] {
+        return router.routes
+    }
+    
+    public init() {
+        self.router = .init()
+    }
+    
+    public func register(route: Route<Responder>) {
+        router.register(route: route)
+    }
+
+    public func route(command: String, on req: Request) -> Responder? {
+        var parameters = Parameters()
+        let responder = router.route(path: command.lowercased().split(separator: " "), parameters: &parameters)
+        return responder
+    }
+}
+
+// Convenience functions
+extension GithubCommandRouter {
+    public func register<T: ResponseEncodable>(
```

Comment:
I'd name this command for readability
```swift
myGithubThing.command("thing") { ... }
```
```suggestion
    public func command<T: ResponseEncodable>(
```